go.sum

```diff
@@ -29,6 +29,10 @@ github.com/hashicorp/go-version v1.6.0 h1:feTTfFNnjP967rlCxM/I9g701jU+RN74YKx2mO
 github.com/hashicorp/go-version v1.6.0/go.mod h1:fltr4n8CU8Ke44wwGCBoEymUuxUHl09ZGVZPK5anwXA=
 github.com/hashicorp/hcl-lang v0.0.0-20221014125844-7eceda07a779 h1:xzrUeFJXnrQ3U7sNrPLdtY8d2kmYr1BqY8X2fCagsl0=
 github.com/hashicorp/hcl-lang v0.0.0-20221014125844-7eceda07a779/go.mod h1:olv+pM633vw0rXvyHIPjw62ur/FNqqNV7b5TfSxLY6E=
+github.com/hashicorp/hcl-lang v0.0.0-20221028094743-e1208e5a6b79 h1:jpXquu60qSM63DVnzfB5PNiVQcSnBiqzhteTf3OirP4=
+github.com/hashicorp/hcl-lang v0.0.0-20221028094743-e1208e5a6b79/go.mod h1:rn5V4b1/658M3g1o+MtdIUl/ZNoCVasB9T31j8XllMM=
+github.com/hashicorp/hcl-lang v0.0.0-20221028130326-b8c8114d3c7e h1:RiqJbM0okTMOxIKZcgKbQwU2L0X8Hwk/0jc3EQQoeMQ=
+github.com/hashicorp/hcl-lang v0.0.0-20221028130326-b8c8114d3c7e/go.mod h1:rn5V4b1/658M3g1o+MtdIUl/ZNoCVasB9T31j8XllMM=
```

Comment:
`go mod tidy` can get rid of the duplicate records here, it's what I'd always run after `go get` and before committing `go.sum`.